admin.go

```diff
@@ -0,0 +1,360 @@
+package sarama
+
+import "errors"
+
+// ClusterAdmin is the administrative client for Kafka, which supports managing and inspecting topics,
+// brokers, configurations and ACLs. The minimum broker version required is 0.10.0.0.
+// Methods with stricter requirements will specify the minimum broker version required.
+// This client was introduced in 0.11.0.0 and the API is still evolving.
+// You MUST call Close() on a client to avoid leaks
+type ClusterAdmin interface {
+	// Creates a new topic .This operation is supported by brokers with version 0.10.1.0 or higher.
+	// It may take several seconds after CreateTopic returns success for all the brokers
+	// to become aware that the topics have been created. During this time, ClusterAdmin#listTopics
+	// and ClusterAdmin#describeTopics may not return information about the new topics.
+	// This operation is supported by brokers with version 0.10.1.0 or higher.
+	// The validateOnly option is supported from version 0.10.2.0.
```

Comment:
the method doesn't take a `validateOnly` flag

Reply:
gave option for  validateOnly flag